central/policy/service/service_impl.go

```diff
@@ -68,6 +71,7 @@ var (
 			"/v1.PolicyService/ExportPolicies",
 			"/v1.PolicyService/PolicyFromSearch",
 			"/v1.PolicyService/GetPolicyMitreVectors",
+			"/v1.PolicyService/SaveAsCustomResources",
```

Comment:
SaveAsCustomResource (singular)?

Reply:
I am following the same naming as other entries in this service. We do support save-as multiple policies.